lms/djangoapps/ccx/views.py

```diff
@@ -179,6 +179,13 @@ def dashboard(request, course, ccx=None):
                 model='CourseEnrollment',
             )
         )
+        # if course licensing is enabled, enrollments and enrollments allowed will be in the context.
+        context['course_licensing_student_list_management'] = (
```

Comment:
short and clear. @anfbermudezme 
```suggestion
        context['pending_ccx_members'] = (
```